cmd/jaeger/ingester-remote-backend-storage.yaml

```
@@ -0,0 +1,32 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [kafka]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+  telemetry:
+    metrics:
+      address: 0.0.0.0:8889
+
+extensions:
+  jaeger_storage:
+    grpc:
+      external-storage:
```

use name that doesn't look like it's part of the config struct
```suggestion
      some-external-storage:
```